timesketch/lib/analyzers/win_crash.py

```diff
@@ -0,0 +1,194 @@
+"""Sketch analyzer plugin for Windows crash artefacts."""
+from __future__ import unicode_literals
+
+import re
+
+from timesketch.lib.analyzers import interface
+from timesketch.lib.analyzers import manager
+
```

Comment:
should be two empty lines before the class declaration

(update: I just made the edits myself)